loc/ui/stats.js

```diff
@@ -142,7 +142,8 @@ export default {
       en: 'Move Spd.',
       fr: 'Vit. Mouv.',
       ru: 'Скорость',
-      pt: 'Velocidade'
+      pt: 'Velocidade',
+      tr: 'Hareket Hızı'
```

Comment:
The translation cannot be more than 2 characters longer than the original English. Please shorten this.